### ISSUE TITLE
Allow to transfer channels between user accounts

### ISSUE THREAD

**Issue by ranjithrajv** [Chocobozzz/PeerTube#4563]
**Describe the problem to be solved**

I have a channel with my old user account. Now I moved to a new user account but would like to manage my old channel with new user account. With current settings, I couldn't perform this operation.

**Describe the solution you would like:**

Allow to transfer channels between user accounts. Where a user can raise a request to move channel to proposed user account, and the proposed user account have the choice to accept or reject the transfer.

If this is anyway a complicated process, atleast allow admins to transfer channels between user accounts. 

**Describe alternatives you have considered**

**Comment by SimplyCorbett** [comment]
What you want to accomplish should be possible by modifying the database. I haven't personally checked into this but every channel should be assigned to a user in the database and modifying that user should switch ownership of the channel.

**Comment by minecraftchest1** [comment]
Its been two years. Has there been any work on this?

**Comment by helgator** [comment]
Its been three years. Has there been any work on this?

**Comment by omadacat** [comment]
its been four years, has there been any work on this?

**Comment by Chocobozzz** [comment]
> its been four years, has there been any work on this?

No, but it's on our roadmap for 2025-2026: https://joinpeertube.org/news/roadmap-2025 (the work will start in 2026)

**Comment by stevespaw** [comment]
I look forward to see where this is going. Our biggest issue with bigger institutions is User/Video management. We end up with people operating really with one user and sharing credentials - not good. But they want and need the experience and features of operating that way. If we could have some sort of "sub-users" on an account that would also be a huge step forward.
FWIW..  Thanks

**Comment by omadacat** [comment]
> > its been four years, has there been any work on this?
> 
> No, but it's on our roadmap for 2025-2026: https://joinpeertube.org/news/roadmap-2025 (the work will start in 2026)

i was just joining in on the funny chain because i was trying to transfer channel data from an account and pooling it to another account and seeing if i was just blind and couldn't see it or it didn't exist

but thank you for actually starting to work on something like this, i think it'll make management and archival of peertube channels easier

**Comment by Chocobozzz** [comment]
Merging this issue in https://github.com/Chocobozzz/PeerTube/issues/1417
